.github/workflows/test.yml

```diff
@@ -24,24 +24,29 @@ jobs:
       fail-fast: false
       matrix:
         ruby: ['2.7', '3.0', '3.1', '3.2', '3.3']
-        gemfile: [rack_2_0, rack_3_0, rails_6_0, rails_6_1, rails_7_0, rails_7_1]
-        integration: [false]
+        gemfile: [Gemfile, gemfiles/rack_2_0.gemfile, gemfiles/rack_3_0.gemfile, gemfiles/rails_6_0.gemfile, gemfiles/rails_6_1.gemfile, gemfiles/rails_7_0.gemfile, gemfiles/rails_7_1.gemfile]
+        specs: ['spec --exclude-pattern=spec/integration/**/*_spec.rb']
         include:
           - ruby: '2.7'
-            gemfile: rack_1_0
+            gemfile: gemfiles/rack_1_0.gemfile
```

Comment:
You'll need to remove this part since rack_1_0 is not in the codebase anymore.

Reply:
Done.